compose.yaml

```diff
@@ -38,16 +38,13 @@ services:
     image: redis:7-alpine
     restart: unless-stopped
   postgres:
-    image: pgautoupgrade/pgautoupgrade:latest
+    image: postgres:16-alpine
     ports:
-      - "15432:5432"
-    # The following turns the DB into less durable, but gains significant performance improvements for the tests run (x3
```

Comment:
Shouldn't we keep this comment?  It's useful information to future developers as to why the settings below are being used.

Reply:
committed it back

Reply:
Thanks. :smile: